manifests/postgresql.pp

```diff
@@ -19,7 +19,7 @@
       postgresql::dbcreate { $foreman::db_name:
         role     => $foreman::db_user,
         password => $foreman::db_password,
-        address  => $foreman::db_server,
+        address  => '127.0.0.1/32'
```

Comment:
Please, place this value where $db_server's default is set: https://github.com/ki0/puppet-foreman/blob/master/manifests/params.pp#L117